src/pattern-library/components/patterns/MoleculePatterns.js

```diff
@@ -8,6 +10,7 @@ import {
 import { IconButton, LabeledButton } from '../../../';
 
 export default function MoleculePatterns() {
+  const [showExample1, setShowExample1] = useState(false);
```

Comment:
"Example1" is a bit vague and could be misleading if you decide to re-order examples in the page. Maybe `showModalExample` or similar?

Reply:
Renamed.